### ISSUE TITLE
Example of passing data from server to client? 

### ISSUE THREAD

**Issue by CDIDEV** [dlwebdev/angular2-mean-slim#1]
Do you have an example in here to pass data from the server to the client?

**Comment by dlwebdev** [comment]
If you're starting a new angular project, I'd recommend take a look at Angular CLI. This project is pretty outdated now. Take a look at my most recent project I'm writing with Angular 4: https://github.com/dlwebdev/angular4-bill-manager/ 

To get data from the server to the client, you'll want to use a service:
https://github.com/dlwebdev/angular4-bill-manager/blob/master/src/app/services/bill.service.ts 

To get data from the database to the service you'll need a new route in express:
https://github.com/dlwebdev/angular4-bill-manager/blob/master/server/routes/bills.js 

To get the data from the service to the client you just use methods you defined in the service in the front end( client):
https://github.com/dlwebdev/angular4-bill-manager/blob/master/src/app/bills/manage/manage.component.ts 

So, to sum it up, the service is a sort of intermediary that sits between the client and the server to send and receive data.

I hope this helps.
